src/main/java/Duke.java

```diff
@@ -23,8 +23,9 @@ public Duke(String fileName) {
             Ui.printLines("File creation failed.");
         }
     }
+    
     public static void main(String[] args) {
-        new Duke("../data/tasks.txt").run();
+        new Duke("src/main/data/tasks.txt").run();
```

Comment:
Could I ask why you choose to have your `tasks.txt` inside your src folder? Alternatively, I've left mine in `{root}/data/duke.txt` as per the tutorial, which I also think makes sense since you want your data file separate from your class files, but let me know your thoughts!